scripts/generate-weights.sh

```diff
@@ -44,7 +44,7 @@ joystream_pallet_benchmark() {
       --chain=dev \
       --steps=$STEPS \
       --repeat=$REPEAT \
-      --execution=wasm \
+      --execution=native \
```

Comment:
Looks like those changes in this file have been accidently commited (this one and the uncommenting of `joystream_pallet_benchmark bounty bounty`)